packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart

```diff
@@ -89,9 +89,16 @@ class ValidateAndroidDomainResult {
   Map<String, List<DomainError>> domainErrors;
 }
 
-class DeepLinksServices {
-  DeepLinksServices(this.client);
-  final Client client;
+class DeepLinksService {
+  DeepLinksService() {
+    client = Client();
+  }
+
+  late Client client;
```

Comment:
can this just be `final client = Client();` and then we can remove the constructor body?